python/lsst/afw/table/schema.cc

```diff
@@ -177,6 +177,11 @@ PYBIND11_PLUGIN(_schema) {
                                            std::string const &,
                                            std::string const &) const) &Schema::join,
                   "a"_a, "b"_a, "c"_a, "d"_a);
+    clsSchema.def("__repr__", [](Schema const & self) {
+        std::ostringstream os;
+        os << self;
+        return os.str();
+    });
```

Comment:
The current policy is "replicate SWIG", which this does. For most classes I agree that repr is more useful if it includes the class. For Schema it doesn't matter so much because the printout is very verbose and obviously a schema.